assets/js/util/tracking/index.test.js

```diff
@@ -85,6 +85,9 @@ describe( 'trackEvent', () => {
 			trackingID: 'UA-12345678-1',
 			activeModules: [],
 			trackingEnabled: true,
+			currentUserRoles: [ 'administrator' ],
+			authenticatedUser: 1,
+			pluginVersion: global.GOOGLESITEKIT_VERSION,
```

Comment:
Let's use an inline value here for the test rather than use the global.
```suggestion
			pluginVersion: '1.2.3',
```